src/pages/docs/chat/moderation/direct/bodyguard.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Bodyguard
+meta_description: "Detect and remove unwanted content in a Chat Room using Bodyguard AI."
+---
+
+[Bodyguard](https://bodyguard.ai/) is a powerful contextual analysis platform that can be used to moderate content in chat rooms.
+
+The Bodyguard rule is a rule applied to chat rooms in Ably Chat which enables you to use Bodyguard's content moderation capabilities to detect and handle inappropriate content before it is published to other users.
```

Comment:
```suggestion
The Bodyguard integration can be applied to chat rooms so that you can use Bodyguard's content moderation capabilities to detect and handle inappropriate content before it's published to other users.
```